### PR TITLE
Feat/news feed

### PR DESCRIPTION
**Pull request by Jeysieeeee**

### CODE REVIEW

File: components/InputField.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { View, Text, TextInput, TouchableOpacity } from 'react-native'
+import styles from '../Style'
+
+type Props = {
```
Comment (edwinpergis) [inline components/InputField.tsx]: add specific name for props

**Review by edwinpergis — CHANGES_REQUESTED** [review]

**Review by edwinpergis — APPROVED** [review]

File: Style.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignSelf: 'center',
+  },
+  TwitterLogo: {
```
Comment (carlofurio-swr) [inline Style.tsx]: Always use camelcase for naming styles in stylesheet. Same goes for the other names below.

File: components/navigation/ButtomTab.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { View, TouchableOpacity } from 'react-native';
+import { MaterialIcons } from '@expo/vector-icons';
+import { Entypo } from '@expo/vector-icons';
+import { Feather } from '@expo/vector-icons';
+import { Ionicons } from '@expo/vector-icons';
+import styles from '../../Style';
+
+export const ButtomTab = () => {
```
Comment (carlofurio-swr) [inline components/navigation/ButtomTab.tsx]: Did you mean BottomTab?

File: screens/auth/Login.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    <View style={styles.loginContainer}>
+      <Image
+        source={{ uri: 'https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/Twitter-logo.svg/1245px-Twitter-logo.svg.png' }}
+        style={styles.TwitterLogo}
+      />
+      <Text style={styles.logintoTwitterStyle}>Log in to Twitter</Text>
+
+      <Input
+        placeholder={'Username'}
+        onChangeText={function (text: string): void {
```
Comment (carlofurio-swr) [inline screens/auth/Login.tsx]: change it to arrow function

File: screens/auth/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+          throw new Error('Function not implemented.');
+        }} />
+      <Input
+        placeholder={'Password'}
+        onChangeText={function (text: string): void {
```
Comment (carlofurio-swr) [inline screens/auth/Login.tsx]: same goes here

**Review by carlofurio-swr — CHANGES_REQUESTED** [review]

File: screens/auth/Login.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      <Image
+        source={{ uri: 'https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/Twitter-logo.svg/1245px-Twitter-logo.svg.png' }}
+        style={styles.twitterLogo}
+      />
+      <Text style={styles.logintoTwitterStyle}>Log in to Twitter</Text>
+
+      <Input
+        placeholder={'Username'}
+        onChangeText={ (text) => {
+          throw new Error('Function not implemented.');
```
Comment (carlofurio-swr) [inline screens/auth/Login.tsx]: remove this throw error, save the value on the states instead and use it as a value

**Review by carlofurio-swr — CHANGES_REQUESTED** [review]

**Review by carlofurio-swr — APPROVED** [review]

**Review by carlofurio-swr — APPROVED** [review]

**Pull request merged**